src/views/components/common/trace-detail-statistics-table.vue

```diff
@@ -0,0 +1,351 @@
+      TraceSpanLogs,
+    },
+    props: ['data', 'traceId', 'showBtnDetail', 'HeaderType'],
+    watch: {
+      data(val, oldVal) {
+        if (!this.data.length) {
+          this.tableData = [];
+          return;
+        }
+        this.tableData = this.formatData(this.changeTree());
+        this.loading = false;
+      },
+    },
+    data() {
+      return {
+        tableData: [],
+        diaplay: true,
+        // segmentId: [],
+        showDetail: false,
+        list: [],
+        currentSpan: [],
+        loading: true,
+      };
+    },
+    methods: {
+      copy,
+      // TODO 统计计算
+      compute(data){
+        const traceData = data[0].children;
+        const map = new Map();
+        //数据转化
+        for (let i=0; i<traceData.length;i++) {
+          const element = traceData[i];
+          // console.log(element);
+          if (map.has(element.endpointName)) {
+            let arr =  map.get(element.endpointName);
+            arr[0].children.push(element);
+            map.set(element.endpointName,arr);
+          }else{
+            let arr = [];
+            arr.push(element);
+            map.set(element.endpointName,arr);
+          }
+        };
+        console.log(map);
+        const result = [];
+
+       for(let value of map.values()){
+          let maxTime = 0;
+          let minTime;
+          let sumTime = 0;
+          let val = value[0].children;
+          let count = val.length;
+          let endpointName;
+          //如果只出现一次,取value[0]
+          if(count == 0){
+            val = value[0];
+            count = 1;
+            let a = val.endTime;
+            let b = val.startTime;
+            let ms = a - b;
+            maxTime = ms;
+            minTime = ms;
+            sumTime = ms;
+            endpointName = val.endpointName;
+
+          } else {
+            //循环计算
+            for (let i = 0; i < val.length;i++) {
+              let element = val[i];
+              let a = element.endTime;
+              let b = element.startTime;
+              let ms = a - b;
+              //默认赋值
+              if(i == 0){
+                endpointName = element.endpointName;
+                maxTime = ms;
+                minTime = ms;
+              }else{
+                if (ms > maxTime){
+                  maxTime = ms;
+                }
+                if (ms < minTime) {
+                  minTime = ms;
+                }
+              }
+              sumTime = sumTime + ms;
+
+            };
+          }
+          let avgTime = count == 0 ? 0 :(sumTime / count);
+          let jsonStr = {
+              'maxTime': maxTime,
+              'minTime': minTime,
+              'avgTime': avgTime,
+              'count': count,
+              'endpointName': endpointName
+              };
+          result.push(jsonStr);
+        };
+        console.log("--- this is result---");
+        console.log(result);
+        console.log("------");
+        return result;
+      },
+      // 给增加层级关系
```

Comment:
Chinese comments are not allowed. @Fine0830 We need to be careful about this, it seems CI doesn't check this. But the main repo CI will check.